.vscode/settings.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Hey Max, this file should be added to your .gitignore. Can you add '.vscode/' (without the quotes) to your .gitignore and push that up?